server/data/quotes.json

```
@@ -335,6 +335,10 @@
     "author": "Mark Amidon",
     "sentence": "Language is the means of getting an idea from my brain into yours without surgery."
   },
+  {
+    "author":"Melina Marchetta",
```

Please add a space between the key and its value as shown:

```suggestion
    "author": "Melina Marchetta",
```